modular_skyrat/modules/modular_items/lewd_items/code/lewd_structures/bdsm_furniture.dm

```diff
@@ -288,8 +287,7 @@
 		to_chat(user, span_notice("You begin unfastening the frame of x-stand..."))
 		if(P.use_tool(src, user, 8 SECONDS, volume=50))
 			to_chat(user, span_notice("You disassemble the x-stand."))
-			var/obj/item/x_stand_kit/C = new
-			C.loc = loc
+			new /obj/item/x_stand_kit(get_turf(user))
```

Comment:
```suggestion
			new /obj/item/x_stand_kit(get_turf(src))
```
perhaps this instead? I don't know, this is just preference at this point, ahaha.